cartography/intel/aws/__init__.py

```diff
@@ -17,9 +17,12 @@
 logger = logging.getLogger(__name__)
 
 
-def _sync_one_account(neo4j_session, boto3_session, account_id, regions, sync_tag, common_job_parameters):
+def _sync_one_account(neo4j_session, boto3_session, account_id, sync_tag, common_job_parameters):
     iam.sync(neo4j_session, boto3_session, account_id, sync_tag, common_job_parameters)
     s3.sync(neo4j_session, boto3_session, account_id, sync_tag, common_job_parameters)
+
+    regions = ec2.get_ec2_regions(boto3_session)
```

Comment:
Let's surround this with the `try` `except` from the original line 93-106 with something like this:

```python
    try:	    
        regions = ec2.get_ec2_regions(boto3_session)	
    except botocore.exceptions.ClientError as e:	
        logger.debug("Error occurred getting EC2 regions.", exc_info=True)	
        logger.error(	
            (	
                "Failed to retrieve AWS region list, an error occurred: %s. Could not get regions for "	
                "account %s."	
            ),	
            e,
            account_id,
        )	
        return
```